src/Bridge/Doctrine/Orm/ItemDataProvider.php

```diff
@@ -32,22 +36,27 @@ class ItemDataProvider implements ItemDataProviderInterface
     private $propertyNameCollectionFactory;
     private $propertyMetadataFactory;
     private $itemExtensions;
-    private $decorated;
+    private $decoratedProvider;
 
     /**
      * @param ManagerRegistry                        $managerRegistry
      * @param PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory
      * @param PropertyMetadataFactoryInterface       $propertyMetadataFactory
      * @param QueryItemExtensionInterface[]          $itemExtensions
-     * @param ItemDataProviderInterface|null         $decorated
+     * @param ItemDataProviderInterface|null         $decoratedProvider
      */
-    public function __construct(ManagerRegistry $managerRegistry, PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory, PropertyMetadataFactoryInterface $propertyMetadataFactory, array $itemExtensions = [], ItemDataProviderInterface $decorated = null)
-    {
+    public function __construct(
+        ManagerRegistry $managerRegistry,
+        PropertyNameCollectionFactoryInterface $propertyNameCollectionFactory,
+        PropertyMetadataFactoryInterface $propertyMetadataFactory,
+        array $itemExtensions = [],
+        ItemDataProviderInterface $decoratedProvider = null
+    ) {
```

Comment:
code style change: can't remember what was our policy here but I'm not found of having to scroll like crazy to see all the arguments... hence the change